src/App.tsx

```diff
@@ -1,109 +1,22 @@
 import React from 'react';
 
 import './App.scss';
-
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import { Post } from './Types/Post';
```

Comment:
Please add this file to your commit and fix this problem
![image](https://user-images.githubusercontent.com/96409563/197714288-a784937c-8ec4-4c75-97ae-6ffbf22a97a9.png)